app/src/main/java/com/glia/exampleapp/Utils.java

```diff
@@ -66,11 +66,15 @@ public static UiTheme getUiThemeByPrefs(SharedPreferences sharedPreferences, Res
         builder.setVisitorMessageTextColor(visitorMessageTextColor);
         builder.setOperatorMessageBackgroundColor(operatorMessageBackgroundColor);
         builder.setOperatorMessageTextColor(operatorMessageTextColor);
+
+        builder.setWhiteLabel(whiteLabel);
+
+        // choice card attributes
+        builder.setChoiceCardContentTextConfiguration(null);
```

Comment:
why null?

Reply:
Same logic as with buttons- there are not Settings options currently defined for this specific ChoiceCard Text Configuration in the Settings view so nothing to pass on here
(You can add them later if you want - prefs with presets mb?)

Reply:
clear, maybe a comment is suitable there?

Reply:
https://glia.atlassian.net/browse/MUIC-626
https://glia.atlassian.net/browse/MUIC-627